### ISSUE TITLE
Push releases to crates.io

### ISSUE THREAD

**Issue by Xophmeister** [tweag/topiary#782]
**Is your feature request related to a problem? Please describe.**
The current GitHub Actions release workflow only creates a GitHub release. New releases aren't available on crates.io.

**Describe the solution you'd like**
Push the release to crates.io automatically.

**Describe alternatives you've considered**
We're doing this manually. This may be prudent for the time being, while the release process is a bit messy...

**Comment by Xophmeister** [comment]
(See also #615)

**Comment by KaKaLuoTeSun** [comment]
好的

**Comment by yannham** [comment]
Is this solved by the current CI setup?

**Comment by Xophmeister** [comment]
No: Releases are still pushed manually when a release is made. I think #856 would need to be completely satisfied before I would feel confident about automatic pushes to crates.io
